WordPress/gutenberg
# Add customize widgets inserter

## Description

Close #29285.

## Known issues
- [x] ~~Clicking on the backdrop of the inserter won't close it (should it?)~~ (it shouldn't)
- [x] Clicking "Browse all" in the quick inserter won't close the quick inserter
- [x] Weird positioning issue in the quick inserter between 2 blocks
- [ ] Unable to add a new block (#30221)
- [x] Overflowing issue in Safari only
- [x] Should move the focus to the searchbox after opening the inserter.

## How has this been tested?



- Enable "Widgets in customizer" in the experiments.
- Go to Appearance -> Customize -> Widgets.
- Try opening the inserter.
- Pressing <kbd>esc</kbd> should close the inserter.

## Screenshots 

https://user-images.githubusercontent.com/7753001/112036262-63690f80-8b7b-11eb-9984-e7d685bc7c2f.mp4

## Types of changes




New feature

## Checklist:
